### ISSUE TITLE
Immediate crash on start

### ISSUE THREAD

**Issue by MightyKnight** [Geek202/TomsServerUtils#18]
Hi Tom,
I'm facing issues starting your mod (it crashes) and can't figure out what it is. I tried to follow the installation instructions as best as possible, hoping this is not an errror on my behalf.

**What happens?**
Server crashes immediately on start, not even the config file for the mod is generated.

I'm using:
- Fabric Loader 0.11.3
- Minecraft 1.16.5
- Java 16

Only having the following three mods installed:
- Fabric API 0.34.2
- Fabric Language Kotlin 1.6.0
- TomsServerUtils 0.6.0

Downloaded the file from Modrinth.

**The following error message is shown in the console:**
```
[23:30:28] [main/FATAL]: Failed to start the minecraft server
java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'toms-server-utils'!
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53) ~[fabric-server-launch.jar:?]
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36) ~[fabric-server-launch.jar:?]
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointServer.start(EntrypointServer.java:32) ~[fabric-server-launch.jar:?]
        at net.minecraft.server.Main.main(Main.java:92) [intermediary-server.jar:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:567) ~[?:?]
        at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226) [fabric-server-launch.jar:?]
        at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146) [fabric-server-launch.jar:?]
        at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:28) [fabric-server-launch.jar:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
        at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78) ~[?:?]
        at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
        at java.lang.reflect.Method.invoke(Method.java:567) ~[?:?]
        at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:62) [fabric-server-launch.jar:?]
        at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:106) [fabric-server-launch.jar:?]
        at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:50) [fabric-server-launch.jar:?]
Caused by: java.lang.NoClassDefFoundError: me/geek/tom/serverutils/libs/org/apache/commons/io/FileUtils
        at me.geek.tom.serverutils.ConfigKt.loadConfig(ConfigKt.java:57) ~[TomsServerTools-0.6.0.jar:?]
        at me.geek.tom.serverutils.ServerUtils2ElectricBoogaloo.onInitialize(ServerUtils2ElectricBoogaloo.java:40) ~[TomsServerTools-0.6.0.jar:?]
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?]
        ... 17 more
Caused by: java.lang.ClassNotFoundException: me.geek.tom.serverutils.libs.org.apache.commons.io.FileUtils
        at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:636) ~[?:?]
        at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:182) ~[?:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
        at net.fabricmc.loader.launch.server.InjectingURLClassLoader.loadClass(InjectingURLClassLoader.java:56) ~[fabric-server-launch.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
        at net.fabricmc.loader.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:168) ~[fabric-server-launch.jar:?]
        at java.lang.ClassLoader.loadClass(ClassLoader.java:519) ~[?:?]
        at me.geek.tom.serverutils.ConfigKt.loadConfig(ConfigKt.java:57) ~[TomsServerTools-0.6.0.jar:?]
        at me.geek.tom.serverutils.ServerUtils2ElectricBoogaloo.onInitialize(ServerUtils2ElectricBoogaloo.java:40) ~[TomsServerTools-0.6.0.jar:?]
        at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50) ~[fabric-server-launch.jar:?]
        ... 17 more
```

**Comment by Tom-The-Geek** [comment]
It seems I messed up the shadow configuration for the apache libraries, so its failing on the step when it uses `FileUtils` to copy out the default config. As a temporary workaround, you should be able to create the `config/serverutils.toml` file manually and fill it with the default config from [here](https://github.com/Geek202/TomsServerUtils/blob/main/src/main/resources/serverutils_default.toml)

**Comment by MightyKnight** [comment]
Thank you, got it running now. 

Although when I tried the workaround first another issue came up:
  `java.lang.NoSuchMethodError: 'double kotlin.time.DurationKt.getMinutes(int)'` (https://pastebin.com/raw/JuBJh49k)
  
Since it was clear it had to do something with Kotlin, I downgraded fabric-language-kotlin from 1.6.0 to 1.5.0, and voilà it worked.
Might be something you want to change for the next release.
